bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -184,6 +187,15 @@ else:
 if tids_with_trigs is not None:
     template_ids = numpy.intersect1d(tids_with_trigs, template_ids)
 
+# Apply cuts as defined on the command line
```

Comment:
not sure why 'as defined on the command line' .. suggest just 'Apply cuts to templates'